omod/src/main/java/org/openmrs/module/locationbasedaccess/web/controller/LBACOnOffController.java

```diff
@@ -0,0 +1,45 @@
+package org.openmrs.module.locationbasedaccess.web.controller;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.openmrs.api.context.Context;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.BaseRestController;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Controller
+@RequestMapping(value = "/rest/" + RestConstants.VERSION_1 + "/lbac/on-off")
+public class LBACOnOffController extends BaseRestController {
+
+	@RequestMapping(method = RequestMethod.POST)
```

Comment:
Can't we use postMapping? 
https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/web/bind/annotation/PostMapping.html

Reply:
@suthagar23  The postMapping Annotation is from 4.3 the one we are using currently is 4.1.4.